lexicons/network/habitat/repo/getBlob.json

```diff
@@ -0,0 +1,36 @@
+{
+  "lexicon": 1,
+  "id": "network.habitat.repo.getBlob",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Get a blob associated with a given account. Returns the full blob as originally uploaded. Does not require auth; implemented by PDS.",
```

Comment:
if we don't wanna write detailed description now let's at least remove the details that aren't true with privi